llvm/lib/Frontend/OpenMP/OMPIRBuilder.cpp

```diff
@@ -145,6 +145,8 @@ static bool isValidWorkshareLoopScheduleType(OMPScheduleType SchedType) {
 }
 #endif
 
+Function *GLOBAL_ReductionFunc = nullptr;
```

Comment:
We should (almost) never have globals flying around. What is going on here?

Reply:
We don't need this for now, I've removed it.